pennylane/ops/functions/is_commuting.py

```diff
@@ -19,41 +19,31 @@
 
 import pennylane as qml
 from pennylane.ops.op_math import Prod, SProd, Sum
-from pennylane.pauli.utils import _wire_map_from_pauli_pair
 
 
-def _pword_is_commuting(pauli_word_1, pauli_word_2, wire_map=None):
+def _pword_is_commuting(pauli_word_1, pauli_word_2):
     r"""Checks if two Pauli words commute.
 
     Args:
-        pauli_word_1 (Observable): first Pauli word in commutator
-        pauli_word_2 (Observable): second Pauli word in commutator
-        wire_map (dict[Union[str, int], int]): dictionary containing all wire labels used in
-            the Pauli word as keys, and unique integer labels as their values
+        pauli_word_1 (Observable): first Pauli word in commutator.
+        pauli_word_2 (Observable): second Pauli word in commutator.
```

Comment:
```suggestion
        pauli_word_1 (Operator): first Pauli word in commutator.
        pauli_word_2 (Operator): second Pauli word in commutator.
```

Might as well update this now.